android/app/src/main/java/com/mattermost/rnbeta/CustomPushNotification.java

```diff
@@ -132,12 +144,12 @@ public void onReceived() {
         final String ackId = initialData.getString("ack_id");
         final String postId = initialData.getString("post_id");
         final String channelId = initialData.getString("channel_id");
+        final String rootId = initialData.getString("root_id");
+        final boolean isCRTEnabled = initialData.getString("is_crt_enabled") != null && initialData.getString("is_crt_enabled").equals("true");
         final boolean isIdLoaded = initialData.getString("id_loaded") != null && initialData.getString("id_loaded").equals("true");
         int notificationId = CustomPushNotificationHelper.MESSAGE_NOTIFICATION_ID;
         if (postId != null) {
             notificationId = postId.hashCode();
-        } else if (channelId != null) {
```

Comment:
Why did you remove this one ?

Reply:
When we went through the code, we discussed about the possible use cases where postId is not available and decided to remove this one.

Reply:
Have you tried on Android the "Send test notification" and validate this does not cause any issues ?

Reply:
Yes it is working. Can I revert the code in case we are unable to recall the edge case where we might need this?

Reply:
Yeah, I don;t really recall which is the case this was needed, perhaps is safest to keep